### ISSUE TITLE
validation loss="nan"

### ISSUE THREAD

**Issue by vanikanjirangat** [yao8839836/text_gcn#138]
While I am trying to run the model, the training loss decreases, while the validation loss is always "nan".
I am trying to run the code in google colab with python 3.8.15 and tensorflow  2.9.2.  I solved the compatibility issues in tensorflow using tf.compat.v1. and had to do tf.compat.v1.disable_eager_execution().

Can you please point out the possible reasons?

Thanks.
